labs/src/main/scala/org/scalalabs/basic/lab01/HelloWorldExercise.scala

```diff
@@ -45,7 +48,7 @@ object HelloWorld {
    * More on method declarations can be found here:
    * http://programming-scala.labs.oreilly.com/ch02.html#MethodDeclarationsAndDefinitions
    */
-	def echo(text: String): String = "FixMe"
+	def echo(text: String): String = "Echo"
```

Comment:
Given that the tests only ever call this function with `"Echo"`, I think the intended definition was `= text`, not `= "Echo"`.